internal/parser/parser_test.go

```diff
@@ -24,20 +25,20 @@ import (
 	policyv1 "github.com/cerbos/cerbos/api/genpb/cerbos/policy/v1"
 	privatev1 "github.com/cerbos/cerbos/api/genpb/cerbos/private/v1"
 	sourcev1 "github.com/cerbos/cerbos/api/genpb/cerbos/source/v1"
-	"github.com/cerbos/cerbos/internal/protoyaml"
+	"github.com/cerbos/cerbos/internal/namer"
+	parser1 "github.com/cerbos/cerbos/internal/parser"
```

Comment:
This is a slightly confusing alias... what about importing the YAML parser as `yamlparser` and letting this one just be `parser`?

Reply:
Indeed. I renamed the package from `protoyaml` to `parser` at the last minute and didn't check too closely what it did to the source 🤦🏽